README.md

```diff
@@ -9,6 +9,8 @@ Given json defining a table, capable of rendering a table in multiple formats
 | Environment variable       | Default                  | Description                                            |
 | -------------------------- | ------------------------ | -----------                                            |
 | BIND_ADDR                  | :23100                   | The host and port to bind to                           |
+| HealthCheckInterval           | Interval between health checks                                                            |    30 seconds |
+| HealthCheckCriticalTimeout    | Amount of time to pass since last healthy health check to be deemed a critical failure    |    90 seconds |
```

Comment:
Should this be caps like the others?

```suggestion
| HEALTHCHECK_CRITICAL_TIMEOUT    | Amount of time to pass since last healthy health check to be deemed a critical failure    |    90 seconds |
```